### PR TITLE
(PC-5807) accessibility block

### PR DESCRIPTION
**Pull request by Giregm**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-5807

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [x] Added new reusable components to the AppComponents page and communicate to the team on slack
- [x] Added a **screenshot** for UI tickets.

## Screenshots

| \*iOS - [Phone name] | \*ios - [Phone name] | android - [Phone name] | Little - [Phone name] |
| -------------------- | :----------------------: | --------------------: | --------------------: |
|         image             |         image      |                image |                       |

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn version:testing` (this will create a commit with a tag)
- then run `git push --follow-tags`

### CODE REVIEW

**Review by antoinewg — APPROVED** [review]
Joli. J'ai qqs petites remarques de forme mais sinon c'est bon pour moi

File: src/features/offer/pages/Offer.tsx [hunk 1]
```diff
@@ -103,6 +112,16 @@ export const Offer: FunctionComponent = () => {
           </AccordionItem>
         </Section>
 
+        <Section visible={shouldDisplayAccessibilityBlock}>
```
[the remark this answers is not in the document]
Comment (Giregm) [reply src/features/offer/pages/Offer.tsx:115]: ça me va !

File: src/features/offer/pages/Offer.tsx [hunk 2]
```diff
@@ -18,10 +18,18 @@ import {
   OfferHero,
   OfferIconCaptions,
   OfferWhereSection,
+  AccessibilityBlock,
+  OfferPartialDescription,
 } from '../components'
-import { OfferPartialDescription } from '../components/OfferPartialDescription'
 import { dehumanizeId } from '../services/dehumanizeId'
 
+const tmpHandicapAccessibilityDate = {
```
[the remark this answers is not in the document]
Comment (Giregm) [reply src/features/offer/pages/Offer.tsx:26]: bonne idée ! j'ai mis le ticket 5804, ticket front "enrichissement du champ"

File: src/features/offer/atoms/AccessibilityAtom.tsx [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        onLayout={(event: LayoutChangeEvent) => {
+          setWidth(event.nativeEvent.layout.width)
+        }}>
+        <Spacer.Flex />
+        <Icon size={getSpacing(12)} />
+        <Spacer.Flex />
+        <ValidationContainer>
+          <ValidationMark
+            isValid={isAccessible}
+            size={getSpacing(6)}
```
[the remark this answers is not in the document]
Comment (Giregm) [reply src/features/offer/atoms/AccessibilityAtom.tsx:41]: bien vu !

**Review by antoinewg — APPROVED** [review]

**Pull request merged**